### ISSUE TITLE
History of direct message converastion disappear

### ISSUE THREAD

**Issue by VasiliyMooduckovich** [RocketChat/Rocket.Chat.Electron#1859]
## My Setup

- Operating System: Windows 10 Pro 1909 (18363.1139)
- App Version: 3.0.5
- Installation type: exe

- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description
Disappearing conversation 
![ZFK7BmkZYy](https://user-images.githubusercontent.com/73686738/97672990-0e2d4400-1a9c-11eb-93a2-c0cd7a69c587.gif)

### Current Behavior
Last direct converstion is disappearing when closing client from system tray icon, but from a web-browser works fine. Deleting rocket.chat profile from appdata helps until next time. We are using ldap auth (just saying).

### Expected Behavior
History of conversation shouldn't going anywhere until user or administrator do so.

**Comment by ankar84** [comment]
I don't see such behavior with 3.0.5 on Windows 10 1809

**Comment by MJPGPleasant** [comment]
I can repo this in 3.0.4. After repo, click on "home", close the app, relaunch, and the missing conversation history will return.

Have not tried to repo on 3.0.5 as of yet.

**Comment by DrWarpMan** [comment]
> I don't see such behavior with 3.0.5 on Windows 10 1809

I've got 3.0.5 and I am also experiencing this bug.

> After repo, click on "home", close the app, relaunch, and the missing conversation history will return.

Thanks for the workaround.

**Comment by aleksandr-kotlyar** [comment]
Not only history of direct messages can disappear, it can happen with any channel. Reloading the page sometimes helps, but sometimes it takes more reloading attempts and restarting application to turn back history. So, the bug is bug. No workaround could cancel it.

**Comment by DrWarpMan** [comment]
> No workaround could cancel it.
Well the workaround didn't cancel it or anything, but it helped me view the messages until it happens again.

**Comment by NotACreativeUsername** [comment]
I will add, I'm on 2.17.11 and get the same issue, workaround works for now for us.

**Comment by MJPGPleasant** [comment]
Can repo in 3.0.7 - possibly related to turning on "Forget User Session on Window Close" (?)

**Comment by Bengrunt** [comment]
Hello,

Same issue with channels, direct messages, whatever.
That's completely random. I've got users experiencing this since we got through with server v3.8.x I'd say.

We're currently running v3.9.1 and ours users are on various platforms and client versions (Win10, Linux, MacOS, Android, IOS).
That'd be great to get this sorted out since it's got a major impact on UX.

So far nothing pops out in the server logs, everything on the MongoDB cluster seems fine as well.

Thanks!
